### PR TITLE
Aita 8 rooms

### PR DESCRIPTION
**Pull request by RioCantre**

### CODE REVIEW

File: src/views/pages/market/styled.ts [hunk 1]
```diff
@@ -3,10 +3,11 @@ import { TitleH1 } from 'views/components/Title';
 import { Wrapper } from 'views/components/Wrapper';
 
 export const ContentWrapper = styled(Wrapper)`
-  height: 77vh;
+  height: 67vh;
```
Comment (VeiveHata) [inline src/views/pages/market/styled.ts]: don`t set height for wrapper, it cause scrolling

File: src/views/pages/market/styled.ts [hunk 2]
```diff
@@ -3,10 +3,11 @@ import { TitleH1 } from 'views/components/Title';
...
   margin: auto;
 `;
 
 export const TitleMarket = styled(TitleH1)`
-  margin-top: 15rem;
+  margin-top: 11rem;
```
Comment (VeiveHata) [inline src/views/pages/market/styled.ts]: use mb prop on component

File: src/views/pages/rooms/components/RoomList/styled.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components';
+import { TitleH2 } from 'views/components/Title';
+import { Button } from 'views/components/Button';
+import { mobileDevice } from 'views/theme/mediaQuery';
+
+export const RoomBaseLayout = styled.div`
+  background-color: ${({ theme }) => theme.colors.backgroundSecondary};
+  height: 60vh;
```
Comment (VeiveHata) [inline src/views/pages/rooms/components/RoomList/styled.ts]: do not set height, it should adoption to any place you put it in. so height is managed by tab content in rooms or it === content height

File: src/views/pages/rooms/index.tsx [hunk 4]
```diff
@@ -1,13 +1,18 @@
...
+import { Hackathon } from '../hackathonNEAR';
 
 const tabs = [
-  { title: 'Main', content: <div>main</div>, id: 'main' },
-  { title: 'Rooms', content: <div>playRrooms</div>, id: 'playRooms' },
-  { title: 'Market', content: <Market />, id: 'market' },
-  { title: 'Achievements', content: <div>Achievements</div>, id: 'achievements' },
+  { title: 'Profile', content: <Hackathon />, id: 'profile' },
+  { title: 'Rooms', content: <RoomsList />, id: 'playRooms' },
+  { title: 'Rating', content: <ContentWrapper>Coming Soon</ContentWrapper>, id: 'rating' },
```
Comment (VeiveHata) [inline src/views/pages/rooms/index.tsx]: if we have this in several places create a separate component for it

File: src/views/pages/rooms/styled.ts [hunk 5]
```diff
@@ -17,3 +17,14 @@ export const RoomsWrapper = styled(Wrapper)`
     }
   }
 `;
+
+export const ContentWrapper = styled.div`
+  text-align: center;
+  background-color: ${({ theme }) => theme.colors.backgroundSecondary};
+  height: 60vh;
```
Comment (VeiveHata) [inline src/views/pages/rooms/styled.ts]: do not set height

**Review by TangerineNe — APPROVED** [review]

**Pull request merged**